tutorials/4 Analysis/C. Parametric sweeps/4.21 Capacitance matrix.ipynb

```diff
@@ -208,42 +180,33 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Pass a dict with arguments to be used add Setup to project.\n",
-    "# If you don't pass all the arguments, the default is determined by \n",
-    "# QQ3DRenderer's default_options.\n",
+    "# For the simulation software, if you don't want default values, \n",
+    "#  update with what you want.\n",
+    "\n",
+    "# If a setup named \"sweeper_q3d_setup\" exists in the project, it will be deleted, \n",
+    "# and a new setup will be added.\n",
+    "c1.sim.setup.name = \"sweeper_q3d_setup\"\n",
     "\n",
-    "# The name of setup will be \"Sweep_q3d_setup\".  \n",
-    "# If a setup named \"Sweep_q3d_setup\" exists in the project, it will be deleted, \n",
-    "# and a new setup will be added with the arguments from setup_args.\n",
-    "q3d_setup_args = Dict(freq_ghz=None,\n",
-    "                      save_fields=None,\n",
-    "                      enabled=None,\n",
-    "                      max_passes=None,\n",
-    "                      min_passes=None,\n",
-    "                      min_converged_passes=None,\n",
-    "                      percent_error=None,\n",
-    "                      percent_refinement=None,\n",
-    "                      auto_increase_solution_order=None,\n",
-    "                      solution_order=None,\n",
-    "                      solver_type=None)\n",
+    "# For q3d, these are the agruments that you can change. freq_ghz=None:\n",
```

Comment:
Best not to have text with the explicit list as the list will change in the future. You can instead just add an executable cell with the line: `c1.sim.setup`, which will output the correct list with all the defaults.

Reply:
> Best not to have text with the explicit list as the list will change in the future. You can instead just add an executable cell with the line: `c1.sim.setup`, which will output the correct list with all the defaults.

Pushed a commit. Used your idea.